packages/webpack/lib/shims/build.js

```diff
@@ -1,8 +1,10 @@
-/* eslint-disable import/no-unresolved */
-/* eslint-disable node/no-missing-import */
-import entryPoint from 'hops/entrypoint';
-/* eslint-enable node/no-missing-import */
-/* eslint-enable import/no-unresolved */
+if (process.env.__HOPS_FAST_BUILD__ === 'true') {
+  require('core-js/stable');
+  require('regenerator-runtime/runtime');
+}
+
+/* eslint-disable-next-line node/no-missing-require */
+const entryPoint = require('hops/entrypoint').default;
```

Comment:
I would like to keep using ESM in the build shim because that makes tree shaking easier. For dev it is fine, because we don't have tree shaking there.
Maybe we could split the shim into two build shims or use the array syntax of webpack entry points (https://webpack.js.org/concepts/entry-points/#single-entry-shorthand-syntax), like:
```js
entry: [require.resolve('core-js/stable'), require.resolve('regenerator-runtime/runtime'), 'require.resolve('../shims/build')],
```